ftp-ballerina/tests/secure_listener_endpoint_test.bal

```diff
@@ -63,8 +63,8 @@ public function testSecureAddedFileCount() {
     // Test fails in 5 minutes if failed to receive watchEvent
     while (timeoutInSeconds > 0) {
         if (secureWatchEventReceived) {
-            log:printInfo("Added file count: " + secureAddedFileCount.toString());
-            test:assertEquals(2, secureAddedFileCount);
+            log:printInfo("Secure added file count: " + secureAddedFileCount.toString());
```

Comment:
```suggestion
            log:printInfo("Secure the added file count: " + secureAddedFileCount.toString());
```

Reply:
This refers to the added file count done by the secure test. So changed accordingly.